### DISCUSSION TITLE
[Suggestion] Mobile Device Gameplay

### DISCUSSION THREAD

**Discussion by sanggusti** [SebLague/Geographical-Adventures#54] [Ideas]
Hi Sebastian, I'm really amazed by this game of yours and I think your game could be more addictive in Mobile devices! Maybe by adding virtual joystick and adjusting build config for mobile devices?

**Comment by SecularSteve** [comment]
I too really want to see a Mobile Edition of this game.

No doubt building the package would be the easy part, but configuring the controls to work on a touchscreen would be another.

As an avid lover of Geography in during most of my school days, this is something I really want to see become more accessible.
